tools/code-analyzer/src/commands/major_minor/index.ts

```diff
@@ -0,0 +1,94 @@
+/**
+ * External dependencies
+ */
+import { CliUx, Command } from '@oclif/core';
+import { join } from 'path';
+import { readFileSync, rmSync } from 'fs';
+import simpleGit from 'simple-git';
+
+/**
+ * Internal dependencies
+ */
+import { cloneRepo } from '../../git';
+
+/**
+ * MajorMinor command class
+ */
+export default class MajorMinor extends Command {
+	/**
+	 * CLI description
+	 */
+	static description = 'Determine major/minor version of WooCommerce plugin';
+
+	/**
+	 * CLI arguments
+	 */
+	static args = [
+		{
+			name: 'branch',
+			description: 'GitHub branch to use to determine version',
```

Comment:
Should we future proof this and require an input for plugin name?

Reply:
@psealock I did consider that. It would require determining the path to the plugin as well, but I don't mind adding that.

Reply:
Resolved in https://github.com/woocommerce/woocommerce/pull/34404/commits/096ab49e6ba91bfe75b90d2d8f4987b6bcca95c6